include/swift/ABI/TaskStatus.h

```diff
@@ -276,6 +276,25 @@ class EscalationNotificationStatusRecord : public TaskStatusRecord {
   }
 };
 
+/// This record signifies that the task has an executor preference.
+/// This preference may be added or removed at runtime, e.g. when multiple `withTaskExecutor { ... }`
+/// blocks are nested, they add more executor preferences.
```

Comment:
Suggestion: Any number of these preferences may be present at runtime, and the
innermost preference takes priority.

Reply:
Good note, added it 👍